test/metabase/api/dashboard_test.clj

```diff
@@ -1992,6 +1992,56 @@
+    (mt/dataset sample-dataset
+      ;; Note that we can directly query the values for the model, but this is
+      ;; nonsensical from a dashboard standpoint as the returned values aren't
+      ;; usable for filtering...
+      (mt/with-temp* [Card [{model-id :id :as native-card}
+                            {:database_id   (mt/id)
+                             :name          "Native Query"
+                             :dataset_query (mt/native-query
+                                             {:query "SELECT category FROM products LIMIT 10;"})
+                             :dataset       true}]]
+        (let [metadata (-> (qp/process-query (:dataset_query native-card))
+                           :data :results_metadata :columns)]
+          (is (seq metadata) "Did not get metadata")
+          (db/update-where! 'Card {:id model-id}
+                            :result_metadata (json/generate-string
+                                              (assoc-in metadata [0 :id]
+                                                        (mt/id :products :category)))))
+        ;; ...so instead we create a question on top of this model (note that
+        ;; metadata must be present on the model) and use the question on the
+        ;; dashboard.
+        (mt/with-temp* [Card [{question-id :id}
+                              {:database_id   (mt/id)
+                               :name          "card on native query"
+                               :dataset_query {:type     :query
+                                               :database (mt/id)
+                                               :query    {:source-table (str "card__" model-id)}}
+                               :dataset       true}]
+                        Dashboard [dashboard {:name       "Dashboard"
+                                              :parameters [{:name      "Native Dropdown"
+                                                            :slug      "native_dropdown"
+                                                            :id        "_NATIVE_CATEGORY_NAME_"
+                                                            :type      :string/=
+                                                            :sectionId "string"}]}]
+                        DashboardCard [_dashcard {:parameter_mappings
+                                                  [{:parameter_id "_NATIVE_CATEGORY_NAME_"
+                                                    :card_id      question-id
+                                                    :target       [:dimension
+                                                                   [:field "CATEGORY" {:base-type :type/Text}]]}]
+                                                  :card_id      question-id
+                                                  :dashboard_id (:id dashboard)}]]
+          (let [url (format "dashboard/%d/params/%s/values" (u/the-id dashboard) "_NATIVE_CATEGORY_NAME_")]
+            (is (= {:values          ["Doohickey"
```

Comment:
```suggestion
            (is (=? {:values          ["Doohickey"
```

super nitty, but I think we should start using this, otherwise every time we update the param values endpoint, we have to update all the tests.